packages/frontend/components/AddCourseModal/SearchResult.tsx

```diff
@@ -48,6 +50,19 @@ export const SearchResult: React.FC<SearchResultProps> = ({
             {course.name}
           </Text>
         </Box>
+        <Box ml="auto" mr="sm"></Box>
+        <a
+          href={getSearchLink(2022, SeasonEnum.FL, course)}
+          target="_blank"
+          rel="noreferrer"
+        >
+          <Image
```

Comment:
just like how the add button uses a chakra-ui icon, can you check if the chakra-ui design system has an info icon?